supplement.csv

```diff
@@ -1,4 +1,8 @@
 Lemma,Sense,Word Class,Year,Quotation,Source,Note
+阿非利加,/,proper noun,1848,?,徐繼畬『瀛環志略』,
+阿非利加洲,/,proper noun,1855,阿非利加州,合信 B. Hobson『博物新編』,Huang lists this source under 阿非利加州 and dates 阿非利加洲 to 1876
+阿弗利加,/,proper noun,1857,?,A. Wylie『六合叢談』,
+埃及,/,proper noun,1607,非洲東北部的一個國家,『幾何原本』,
```

Comment:
```suggestion
```
Again, let’s remove this one. But keep a copy of all the proper nouns you’ve done so far. I’ll get them saved in a separate table in the repo.

Reply:
Understood. I will remove all the proper nouns so far and put them in a separate place.